mobile/README.md

```diff
@@ -0,0 +1,220 @@
+This project was bootstrapped with [Create React Native App](https://github.com/react-community/create-react-native-app).
```

Comment:
Can we please rename this file to lowercase `readme.md` for consistency, and really simplify it so be similar to the other directory readmes? Perhaps we could link to Create React Native App docs somewhere so we don't have to repeat it all here.